### DISCUSSION TITLE
Let us donate money

### DISCUSSION THREAD

**Discussion by CestQuoiUnPseudo** [HttpRafa/modflared#29] [Ideas]
Your mod is the single most important mod in my minecraft server, and I would like to show my thanks by paying your next coffee. Do you have a link to where I could send you some euros?

If you don't feel like it, then please at least take my most heartfelt thanks. Since I have next to no programming experience, I could not have made a server for my friends without your work

**Comment by HttpRafa** [comment]
Thank you so much for your kind words and for using my mod on your Minecraft server!

I'm really glad you're enjoying the mod, and honestly, your appreciation and enjoyment of it are more than enough "payment" for me. I'm just doing this for fun and the love this hobby, so seeing others like you enjoy it is what keeps me going.

**Reply by carterlikespie** [reply]
I'm with this guy from 2 years ago. Just found your mod and it made my server with friends possible and easy without having to pay a huge monthly cost to a server host. You're an absolute beast for making this mod for such a specific use case!
